src/stories/Examples/ResponsiveLoginPage/ResponsiveLoginPage.tsx

```diff
@@ -0,0 +1,114 @@
+import {Box, Grid, Theme, Typography, useMediaQuery} from '@mui/material';
+import arcLogo from 'Assets/ARC_logo.png';
+import heroLogo from 'Assets/hero.jpg';
+import Button from 'Components/Button';
+import Form from 'Components/Forms/Form';
+import FormInput from 'Components/Forms/FormInput';
+import FormPasswordInput from 'Components/Forms/FormPasswordInput';
+import {FormikHelpers} from 'formik';
+import {useState} from 'react';
+import * as yup from 'yup';
+
+type LoginForm = {
+  username: string;
+  password: string;
+};
+
+const initialValues = {
+  username: '',
+  password: '',
+};
+
+const validationSchema = yup.object({
+  username: yup.string().required().label('UserName'),
+  password: yup.string().required().label('Password'),
+});
+
+interface ResponsiveLoginPageProps {}
+
+export const ResponsiveLoginPage: React.FC<ResponsiveLoginPageProps> = () => {
+  const isSmallScreen = useMediaQuery((theme: Theme) => theme.breakpoints.down('md'));
+
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  const handleSubmit = async (values: LoginForm, {setSubmitting}: FormikHelpers<LoginForm>) => {
+    setIsLoading(true);
+    setTimeout(() => {
+      setIsLoading(false);
+      alert(JSON.stringify(values));
+      setSubmitting(false);
+    }, 2000);
+  };
```

Comment:
Is there an alternate of showing values submitted than alerting?
If not let's just toast a success message without this alert.